### PR TITLE
Remember selected vault

### PR DESCRIPTION
**Pull request by rcoderdev**

### CODE REVIEW

File: frontend/src/lib/ui/state/LocalStorage.ts
Comment (enriquesouza) [inline frontend/src/lib/ui/state/LocalStorage.ts:1]: We are already using IndexedDB and also SQL Lite, what are we persisting on Local Storage?

**Review by enriquesouza — COMMENTED** [review]
Please, can you explain why are we also using local storage?

File: frontend/src/lib/ui/state/LocalStorage.ts
Comment (rcoderdev) [reply frontend/src/lib/ui/state/LocalStorage.ts:1]: IndexedDB and SQLite would be a bit of an overkill for simple things that we need to persist, such as in this scenario – the ID of the currently selected vault.

**Review by enriquesouza — APPROVED** [review]
OK, LGTM

**Pull request merged**